lib/telegraf/telegraf.conf.erb

```diff
@@ -113,31 +126,73 @@
   ## Uncomment the following line if you do not need disk serial numbers.
   # skip_serial_number = true
 
+  ## These lines fix the system metric names to basically be
+  ## system.<host>.<measurement>.<field>
+  ## Look at the output template for more info.
+  name_override = "system"
+  [inputs.diskio.tags]
+    system_measurement_tag = "diskio"
+
 
 # Get kernel statistics from /proc/stat
 [[inputs.kernel]]
   # no configuration
 
+  ## These lines fix the system metric names to basically be
+  ## system.<host>.<measurement>.<field>
+  ## Look at the output template for more info.
+  name_override = "system"
+  [inputs.kernel.tags]
+    system_measurement_tag = "kernel"
+
 
 # Read metrics about memory usage
 [[inputs.mem]]
   # no configuration
 
+  ## These lines fix the system metric names to basically be
+  ## system.<host>.<measurement>.<field>
+  ## Look at the output template for more info.
+  name_override = "system"
+  [inputs.mem.tags]
+    system_measurement_tag = "mem"
```

Comment:
Is it worth expanding this to "memory" to match the mostly other unabbreviated ones?